tools/cordova/project.js

```diff
@@ -461,6 +494,41 @@ from Cordova project`, async () => {
     });
   }
 
+  patchCordova() {
+    this.runCommands(`patching Cordova project`, async () => {
+      var cordovaCommonPatch = files.pathJoin(__dirname, 'patches', 'cordova-common@FileUpdater.js');
+      var cordovaCommonFilePath = files.pathJoin(this.projectRoot, 'node_modules/cordova-common/src/FileUpdater.js');
+      files.copyFile(cordovaCommonPatch, cordovaCommonFilePath);
+    });
+  }
+
+  async applyCordovaPatchWhen(cmd) {
+    var stdoutBuffer = "";
+    var stderrBuffer = "";
+    var buffer = captureLogBuffer(stdoutBuffer, stderrBuffer);
+    this.patchCordova();
+    try {
+      await cmd();
+    } catch(e) {
+      if (e.message.includes('fs.cpSync is not a function')) {
+        try {
+          captureLogBuffer();
+          this.patchCordova();
+          await cmd();
+        } catch(e) {
+          // ignore
+        }
```

Comment:
Could we log this error? if not, why can't we log it? Is it noise when it fails on this `catch branch`?

Reply:
Going to test further, but it showed an error saying it was already added instead of simply confirming. Maybe I can ignore the "already added" error and accept every other error that can happen.